app/assets/stylesheets/home.scss

```diff
@@ -1,3 +1,43 @@
 // Place all the styles related to the home controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+.welcome {
+	text-align: center;
+}
+
+.image-size {
+  height: 300px;
+  width: 100%;
+}
+
+.home-image {
+	padding-left: 0px !important;
+	padding-right: 0px !important;
+	border: 3px solid var(--gold-hl);
```

Comment:
White borders for consistency

Reply:
also, remove this file and put it in photos.scss since we're using photos#home as our homepage now.

Reply:
also remove unnecessary javascript